shelf_router/lib/src/router.dart

```diff
@@ -163,6 +163,14 @@ class Router {
   ///
   /// This method allows a Router instance to be a [Handler].
   Future<Response> call(Request request) async {
+    bool _isNotDefaultNotFound(Response response) {
+      return response.statusCode != routeNotFound.statusCode ||
```

Comment:
Is there a better way to determine whether the response is the default without having a static response?